tock/hours/tests/test_forms.py

```diff
@@ -164,16 +172,25 @@ def test_timecard_inline_formset_save_only(self):
         formset.save_only = True
         self.assertTrue(formset.save_only)
 
-    def test_timecard_is_not_100(self):
-        """ Test timecard form data that doesn't total the set working
-        hours """
+    def test_timecard_is_too_big(self):
+        """ Test timecard form data that exceeds the maximum hours allowed """
```

Comment:
i am seeing a total of 92 actually. see ln 151 (22 hours) + ln 153 (20 hours) + ln 179 (50 hours). the `test_timecard_is_too_big` function pulls the base data from `setUp` (which holds two time card objects) and then adds an additional time card object to it, for a total of three time card objects (see ln 180). this make sense?